src/api/key.h

```diff
@@ -0,0 +1,224 @@
+/*
+    This file is part of libnspire.
+
+    libnspire is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    libnspire is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libnspire.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef NSP_KEY_H
+#define NSP_KEY_H
+
+#include <inttypes.h>
+#include "handle.h"
+
+enum nspire_keys {
+    KEYNSP_ESC = 0x1B9600,
```

Comment:
The enum is called `nspire_key(s)`, so for consistency this should be `NSPKEY_` or `NSP_KEY_`, especially because those are in the global namespace